addons/web/static/tests/views/form/form_compiler_tests.js

```diff
@@ -32,7 +32,7 @@ QUnit.module("Form Compiler", () => {
         const arch = /*xml*/ `<form><div>lol</div></form>`;
         const expected = /*xml*/ `
             <t>
-                <div t-att-class="props.class" t-attf-class="{{props.record.isInEdition ? 'o_form_editable' : 'o_form_readonly'}}" class="o_form_nosheet" t-ref="compiled_view_root">
+                <div t-att-class="props.class" t-attf-class="{{props.record.isInEdition ? 'o_form_editable' : 'o_form_readonly'}} d-flex {{ uiService.size &lt; 6 ? &quot;flex-column&quot; : &quot;flex-nowrap h-100&quot; }}" class="o_form_nosheet" t-ref="compiled_view_root">
```

Comment:
any reason for the manual escaping of the quotes instead of single quotes here?

Reply:
Indeed this is generated, I just matched the result to make the assert.